lib/lightning/runtime/runtime_manager.ex

```diff
@@ -44,7 +44,8 @@ defmodule Lightning.Runtime.RuntimeManager do
               port: 2222,
               repo_dir: nil,
               worker_secret: nil,
-              ws_url: "ws://localhost:4000/worker"
+              ws_url: "ws://localhost:4000/worker",
```

Comment:
Quick note that both these env vars will fail if users try to run lightning on a port other than 4000! Can someone fix this for me?

Reply:
Done! 👍